ts/src/binance.ts

```diff
@@ -1071,14 +1071,267 @@ export default class binance extends Exchange {
                     'CMFUTURE': 'inverse',
                     'OPTION': 'option',
                 },
+                'useUnifiedNetworkCodes': false, // temporarily disable this to avoid immediate breaking changes. Soon after announcement we would enable this by default
```

Comment:
I am not sure if we need this option at all, and maybe directly turn on this feature.
but, as binance is too much heavily used, I still added that option to take care & avoid breaking changes for some users

for example, returned response from `fetchDepositAddress`  till date included the the raw `network: 'RAW_NETWORK_ID'` from now there will be unified network code for almost all chains. and idk, if users have logic like
`if (depositNetwork === 'ArbITR()m')` then their codes would be broken. 

decide as you think